### PR TITLE
Update six version to latest

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixe https://github.com/mitodl/micromasters/issues/4150#issuecomment-433386101

#### What's this PR do?
it upgrade six. Which is essential for django 2.1.2 upgrade

#### Where should the reviewer start?
see everything works fine

@pdpinch

### CODE REVIEW

File: edx_api/__init__.py [hunk 1]
```diff
@@ -3,5 +3,5 @@
 edX python REST API client
 """
 
-__version__ = '0.5.0'
+__version__ = '0.5.1'
```
Comment (annagav) [inline edx_api/__init__.py]: Could you undo this change? This is not a release PR...

**Comment by amir-qayyum-khan** [comment]
sure i will do when i reach home
